spec/mini_max_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative "../app/mini_max"
+
+RSpec.describe MiniMax do
+  let(:passcode) {
+    %w(RED RED GREEN GREEN)
+  }
+  let(:colors) {
+    %w(RED ORANGE YELLOW GREEN BLUE PURPLE)
+  }
+
+  describe "#initialize" do
```

Comment:
This should be changed to `describe "#play" do`, since that's the method that gets tested. FYI this code clearly shows that TDD was not involved here, as this implementation of the MiniMax would do exactly the same:
```ruby
class MiniMax
  def initialize(passcode:, combinator:)
    @passcode = passcode
  end

  def play
    @passcode
  end
end
```